toolchain/check/member_access.cpp

```diff
@@ -461,13 +461,13 @@ auto PerformMemberAccess(Context& context, SemIR::LocId loc_id,
   if (required) {
     return HandleAction<SemIR::AccessMemberAction>(
         context, loc_id,
-        {.type_id = SemIR::InstType::TypeId,
+        {.type_id = GetSingletonType(context, SemIR::InstType::TypeInstId),
```

Comment:
I wonder if the constants should be `IncompleteTypeInstId` and `IncompleteTypeId` for the ones that we don't want to use it directly. Which is then a bigger pain to write, so maybe it should also be `GetSingletonType<SemIR::InstType>(context)` to avoid spelling it out/using the incomplete thing at all.